### DISCUSSION TITLE
writing/2024/04/29/losing-my-hands/

### DISCUSSION THREAD

**Discussion by giscus** [jxnl/blog#81] [General]
# writing/2024/04/29/losing-my-hands/

A personal journey exploring the struggle of a machine learning engineer as he faces the consequences of overwork and injury during the pandemic.

https://jxnl.co/writing/2024/04/29/losing-my-hands/

**Comment by blake41** [comment]
I suffered through RSI as well for many years. I'd guess that I've researched this topic more than anyone on the planet. I've benefitted tremendously from all you've shared over the last six months since I started following you on twitter. I'd love to help. I have a lot of suggestions for things that actually work although with things like this YMMV. I cured myself after years of pain and hopelessness using a tendonitis protocol. I'm blake41 at gmail

**Comment by DougDroogSharp** [comment]
Hi, Jason.
I've been bedridden after a long haul Covid crash for the last seven months. 
My tremors have become so bad that GUIs are torture.
I'm a lifelong software developer who wrote 2 best selling games in the 80s. I worked in Microsoft research‘s virtual world group in the 90s as a coder and manager.

I haven't been able to program since 1997 when I started having out of control seizures.

I've always followed AI. I wrote a couple of science fiction novels 10 years ago that advocated open source AI development: Channel Zilch and Hel’s Bet. Great reviews but tiny sales. Available as audiobooks on Amazon:

https://www.amazon.com/Channel-Zilch-Hels-Bet-Book-ebook/dp/B00RC9WYQU

When writing fiction became too painful I discovered that I can draw without much pain so I started doing comics: http://dougsharpcomics.com

Vibecoding has freeD me to make games again. When open claw hit I jumped in right away and had a blast vibecoding a game editor.

As my tremors grew worse I realized I needed something that would become my current project BIRO, voice to text with the built-a built-in capable secretary.

I realized that a lot of handicap people could use the tools that I was building to do their art so that's One Direction I'm going in.

As I adapt my game creation tool to my abilities I'm working on a game called“Humans vs Billionaires”which is in the idea stage.

The stress society rings very very true.

When writing fiction became too painful I discovered that I can draw without much pain so I started doing comics: http://dougsharpcomics.com

When open claw hit I jumped in right away and had a blast vibecoding a game editor.

As my tremors grew worse I realized I needed something that would become my current project BIRO, voice to text with the built-a built-in capable secretary.

I realized that a lot of handicap people could use the tools that I was building to do their art so that's One Direction I'm going in.

As I adapt my game creation tool to my abilities I'm working on a game called“Humans vs Billionaires”which is in the idea stage now.

Five years ago I put together a team to reboot my first best seller, ChipWits, and we've produced a great game that is up on steam. We created a harness so AI could play the game and a few have accepted the challenge including your friend Didier Lopes. He pointed out this post and said I should talk to you and after reading the post I would love to talk to you. 


Excuse voice to text errors because apple voice to text is a hard beast to master

Due to my brain damage I have trouble putting anything in order, which is one of the reasons why I can no longer program. I know this reply is a scrambled hodgepodge but skip over the repetitions.

**Comment by DougDroogSharp** [comment]
Your friend Didier Lopes trained an AI to beat ChipWits and has written great substack article on on it.

He pointed me to you and to this post.
